jog_arm/src/jog_arm/jog_arm_server.cpp

```diff
@@ -573,10 +573,10 @@ void JogCalcs::lowPassFilterPositions() {
 void JogCalcs::lowPassFilterVelocities(const Eigen::VectorXd &joint_vel) {
   for (size_t i = 0; i < jt_state_.name.size(); ++i) {
     jt_state_.velocity[i] =
-        velocity_filters_[i].filter(joint_vel[static_cast<long>(i)]);
+        velocity_filters_[i].filter(joint_vel[i]);
```

Comment:
Indeed, these static casts aren't necessary, but machinekoder added them for a reason. See https://github.com/UTNuclearRoboticsPublic/jog_arm/pull/10